src/encoded/static/components/browse/components/above-table-controls/SelectedFilesDownloadButton.js

```diff
@@ -111,7 +111,7 @@ export class SelectedFilesDownloadButton extends React.PureComponent {
     render(){
         const {
             selectedFiles, filenamePrefix, children, disabled,
-            windowWidth, context, analyticsAddFilesToCart, action,
+            windowWidth, context, analyticsAddFilesToCart, action, session,
```

Comment:
Codacy found an issue: ['windowWidth' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6414303)